packages/browser/src/lib/get-global.ts

```diff
@@ -0,0 +1,16 @@
+// This an imperfect polyfill for globalThis
+export const getGlobal = () => {
```

Comment:
With this, I wonder - can we turn off webpack polyfilling `global` then? That might save us some bytes on the UMD build.
https://webpack.js.org/configuration/node#node

Reply:
Our webpack build is only targeting browser -- there's no extra polyfill.

Reply:
> Our webpack build is only targeting browser -- there's no extra polyfill.

I'm not sure I follow. If you look at the current standalone build you'll find that webpack has polyfilled the global object for us. You've created a new `getGlobal()` function that accomplishes the same goal - but since it references `global` itself I would expect webpack to still polyfill `global` when creating the UMD bundle.

Reply:
Maybe you can show me where the actual polyfill is happening? I'm still not seeing it.

Reply:
Would this get compiled even when I use the `npm` version of the package?